radsat-sk/operation/message/protobuf/RProtobuf.c

```diff
@@ -56,5 +58,5 @@ int protoDecode(uint8_t* incomingBuffer, radsat_message* decodedMessage) {
 	// decode the message into the empty message struct
 	uint8_t success = pb_decode(&stream, radsat_message_fields, decodedMessage);
 
-	return !success;

```

Comment:
Why was this "return !success" ?. I'm sure that with just "return success" (no "!"), the function is working.